### DISCUSSION TITLE
SolaX X1 Hybrid discharge battery to grid

### DISCUSSION THREAD

**Discussion by M0LTE** [wills106/homeassistant-solax-modbus#417] [Q&A]
Hi

Looking to discharge my battery to grid due to arbitrage available with, and encouraged by, my supplier.

If I set export limit to say 3000W, change to Manual Mode, change Manual Mode to Force Discharge, I only get a small amount of export to grid- I assume solar input minus current house load.

Is anyone aware of any other mode of power control available perhaps via modbus I can use to force the inverter to discharge the battery to the grid at a higher rate?

Thanks
Tom

**Comment by wills106** [comment]
Have you had a look at what's described in the Wiki?
[Modbus-Power-Control](https://github.com/wills106/homeassistant-solax-modbus/wiki/Modbus-Power-Control-(remotecontrol-power))

**Reply by M0LTE** [reply]
That looks to have been updated since I last saw it, thank you, I'll check it out.
